ui/src/app/queries.ts

```diff
@@ -20,4 +20,24 @@ export async function postCSR(csr: string) {
         throw new Error('Network response was not ok')
     }
     return response.json()
+}
+
+export async function deleteCSR(id: string) {
+    const response = await fetch("/api/v1/certificate_requests/" + id, {
+        method: 'delete',
+    })
+    if (!response.ok) {
+        throw new Error('Network response was not ok')
+    }
+    return response.json()
+}
```

Comment:
Is this expected to work? The line remains in the table after clicking on the delete button in the modal

Reply:
yes. fixed.